packages/odyssey-react/src/components/Box/Box.tsx

```diff
@@ -303,9 +304,9 @@ export interface BoxProps {
   color?: "body" | "inherit" | false;
   /**
    * Whether to apply the normal font weight class
-   * @default regular
+   * @default normal
    */
-  fontWeight?: "regular" | "inherit" | false;
+  fontWeight?: "normal" | "inherit" | false;
```

Comment:
I'm making this change here to align with the DOM API and be more consistant with other props across this interface to increase maintainability and developer experience.

Incidentally, we make use of a number of adjectives across our system to describe the idea of the standard option within a set: Normal, Default, Regular, Base. In some cases we want to align with DOM APIs so `normal` makes good sense, in other cases we are free from those constraints and use a little bit of everything. Something to keep an eye on moving forward

Reply:
I'd prefer this wherever possible as well. Thanks for calling this out.